src/Agent/NewRelic/Agent/Core/Config/Configuration.cs

```diff
@@ -4907,11 +4907,14 @@ public partial class configurationLogSending
         
         private configurationLogSendingMetrics metricsField;
         
+        private configurationLogSendingForwarding forwardingField;
```

Comment:
This name is a little confusing to me, but I'm ok with it since it is only used internally

Reply:
This is an auto-generated field from Xsd2Code so we can't really change it.